doc/source/examining/loading_data.rst

```diff
@@ -1951,6 +1963,62 @@ have assumed your data is stored in the three-dimensional array
 * Some functions may behave oddly or not work at all.
 * Data must already reside in memory.
 
+.. _loading-stretched-grids:
+
+Stretched Grid Data
+-------------------
+
+.. warning::
+
+   API consistency for loading stretched grids is not guaranteed until at least
+   yt 4.2!  There may be changes in between then and now, as this is a
+   preliminary feature.
+
+With version 4.1, yt has the ability to specify cell widths for grids.  This
+allows situations where a grid has a functional form for its widths, or where
+widths are provided in advance.
+
+.. note::
+
+   At present, support is available for a single grid with varying cell-widths,
+   loaded through the stream handler.  Future versions of yt will have more
+   complete and flexible support!
+
+To load a stretched grid, you use the standard (and now rather-poorly named)
```

Comment:
so we don't advertise how to plug this into a frontend for now, right ? I'm okay with this, since we know it's unstable, and I will gladly have yt_idefix be the avant-guard here, just checking we're on the same page

Reply:
I'm happy to add this once we get AMR support working, I think.  But yeah for now let's not?